.github/workflows/cicd.yml

```diff
@@ -73,7 +76,3 @@ jobs:
         env:
           IMAGE_TAG: ${{needs.build_and_publish.outputs.image_tag}}
           ENVIRONMENT: staging
-          ARM_CLIENT_ID: ${{ fromJSON(secrets.AZURE_CREDENTIALS).clientId }}
-          ARM_CLIENT_SECRET: ${{ fromJSON(secrets.AZURE_CREDENTIALS).clientSecret }}
-          ARM_SUBSCRIPTION_ID: ${{ fromJSON(secrets.AZURE_CREDENTIALS).subscriptionId }}
```

Comment:
I think `ARM_SUBSCRIPTION_ID` might still be necessary during deploy as it's used in `bin/kv_*_ip`.

Reply:
That's right, added it back.